main.js

```diff
@@ -25,12 +25,33 @@ class Car {
         this.$img.classList.remove('east', 'north', 'south')
     }
   }
+  move() {
+    switch (this.direction) {
+      case 'north':
+        this.location[1] -= this.speed
+        break
+      case 'south':
+        this.location[1] += this.speed
+        break
+      case 'east':
+        this.location[0] += this.speed
+        break
+      case 'west':
+        this.location[0] -= this.speed
+        break
+    }
+    this.$img.setAttribute('style', 'top:' + this.location[1] + 'px; left:' + this.location[0] + 'px;')
+  }
+  start() {
+    var drive = car.move.bind(car)
```

Comment:
Instead of binding to a specific car instance, bind to "this" so it works with any car instance.